src/Assets/AssetContainerRepository.php

```diff
@@ -43,7 +43,7 @@ public function make(string $handle = null): AssetContainerContract
 
     public function save(AssetContainerContract $container)
     {
-        $container->toModel()->save();
+        $container->save();
```

Comment:
Did you test this change? Seems like something that was there on purpose.

Reply:
Yes, this is one of the reasons why the unit test failed - also AssetContainer is the one triggering the `AssetContainerSaved` event